es6-shim.js

```diff
@@ -285,6 +292,11 @@
       return typeof x === 'function' && _toString(x) === '[object Function]';
     },
 
+    IsConstructor: function (x) {
+      // We can't tell callables from constructors in ES5
+      return ES.IsCallable(x);
```

Comment:
we could do a "is not arrow function" check here too if we want, but that may not help much